tests/unit/amo/sagas/test_home.js

```diff
@@ -171,7 +172,43 @@ describe(__filename, () => {
       expect(expectedAction).toEqual(loadAction);
     });
 
-    it('does not fetch featured themes if includeFeaturedThemes is false', async () => {
+    it('does not fetch featured themes if includeTrendingExtensions is false', async () => {
```

Comment:
I believe this should be "does not fetch *trending extensions* if includeTrendingExtensions is false".